pkg/storage/storage.go

```diff
@@ -27,6 +27,8 @@ import (
 	"helm.sh/helm/pkg/storage/driver"
 )
 
+const HelmStorageType = "sh.helm.release.v1"
```

Comment:
what does the `v1` mean in this case?

Reply:
it was implemented as per @mattfarina comments in https://github.com/helm/helm/issues/6435#issuecomment-533591950

Reply:
I'm asking because it's not immediately apparent what the significance of that number is. Why does it symbolize? When is it incremented?

If it's intended as a versioning scheme for the release name, then there should be a comment explaining the significance of that storage type.

Reply:
@hickeyma can you please update PR with the comments, thanks

Reply:
@bacongobbler This is a good point especially from a maintenance perspective. I have added comments to the code. Let me know if you need any more clarification.

Reply:
Thank you for the clarification, @mattfarina, and thank you for writing it down, @hickeyma!

Reply:
Not a bother. FYI, I didn't bother responding in the PR as I hoped the description in the code comments would provide the relevant information.